project/Build.scala

```diff
@@ -0,0 +1,44 @@
+import sbt._
+import sbt.Keys._
+import sbt.internal.BuildDef
+
+import net.vonbuchholtz.sbt.dependencycheck.DependencyCheckPlugin.autoImport.dependencyCheckFailBuildOnCVSS
+
+object Build extends BuildDef with BuildCommon {
```

Comment:
Is there any reason you are using a `Build.scala` file? `Build.scala` is deprecated in preference of a simple `build.sbt` file.

Reply:
It's because I wanted to generate the projects dynamically. Using `build.sbt`, I was only able to define them manually, one by one.